blog/content/post/2024-08-19-secureboot-freebsd.md

```diff
@@ -0,0 +1,397 @@
+---
+title: Secure Booting FreeBSD with Dasharo firmware
+abstract: 'Abstract first sentence.
+          Abstract second sentence.
+          Abstract third sentence.'
+cover: /covers/image-file.png
+author: filip.lewinski
+layout: post
+published: true
+date: 2024-10-14
+archives: "2024"
+
+tags:
+  - bsd
+  - dasharo
+  - edk2
+  - secure-boot
+  - protectli
+  - uefi
+categories:
+  - Firmware
+  - IoT
+  - Miscellaneous
+  - OS Dev
+  - App Dev
+  - Security
+  - Manufacturing
+
+---
+
+## Introduction
+
+## UEFI Secure Boot
+
+UEFI Secure Boot is a security feature designed to ensure that only trusted,
+signed software can execute during the boot process, protecting against malware
+and unauthorized code execution. It verifies the signatures of bootloaders and
+kernel binaries against trusted certificates embedded in the firmware.
+
+### Linux
+
+Most distributions took the approach of using a small, signed bootloader called
```

Comment:
Currently, mainstream distributions like Ubuntu or Fedora use that approach, but more minors need help to afford it. So, if you count all distributions, it would not be the most.

Which distributions has Shim signed? https://github.com/rhboot/shim-review/issues?q=is%3Aissue+is%3Aclosed

Reply:
[Corrected](https://github.com/3mdeb/news-and-ideas/pull/603/commits/469cc84541acf2428e46af01de19cf63b5bb0466#diff-1e142bc7519e97361fc0d3867abcb7fa3d257a4b5ccb79795e5e00acebf7d22fR43)